manifests/params.pp

```diff
@@ -290,7 +290,7 @@
   $dns_interface          = 'eth0'
   $dns_zone               = $::domain
   $dns_realm              = upcase($dns_zone)
-  $dns_reverse            = '100.168.192.in-addr.arpa'
+  $dns_reverse            = undef
```

Comment:
Also update the documented `Variant` data type to add `Undef` as a permitted value.